README.md

```diff
@@ -32,32 +32,80 @@ import \
 Import remote ROS .msg files to Capella model layer's root data package:
 
 ```sh
-python -m capella_ros_tools \
-import \
+capella-ros-tools import \
 -i git+https://github.com/DSD-DBS/dsd-ros-msg-definitions-oss \
 -m tests/data/empty_project_60 \
 -l la
 ```
 
-Export local Capella model layer's root data package as ROS .msg files:
+## Export of Capella Classes as ROS2 Messages
+Please mind: If classes don't follow the ROS2 naming conventions, their names as well as property, enumeration value and
+package names will be converted accordingly.
+### Export by layer
+Export local Capella model layer's root data package as ROS .msg files. All msg files will be exported in a single
+package:
 
 ```sh
-python -m capella_ros_tools \
-export \
+capella-ros-tools export \
 -m tests/data/melody_model_60 \
 -l la \
 -o tests/data/melody_msgs
 ```
-
-Export remote Capella model layer's root data package as ROS .msg files:
+Export remote Capella model layer's root data package as ROS .msg files. All msg files will be exported in a single
+package:
 
 ```sh
-python -m capella_ros_tools \
-export \
+capella-ros-tools export \
 -m git+https://github.com/DSD-DBS/coffee-machine \
 -l sa \
 -o tests/data/coffee_msgs
 ```
+### Custom Export
+Use the custom exporter using a config file:
+````yaml
+packages:
+    asdf: <capella_pkg_uuid>
+built_ins:
+    ros_pkg: <capella_build_in_pkg_uuid>
+custom_pkg:
+    "abc":
+      - <capella_cls1_uuid>
+    "xyz":
+      - <capella_cls2_uuid>
+custom_types:
+    Bitset16: int16
+    Bitset32: int32
+````
```

Comment:
The world needs more TOML.

Advantages over YAML:

- `tomllib` is in stdlib since Python 3.11 (previously `tomli` on PyPI)
- very simple and very explicit format
- no significant whitespace
- [it's not YAML](https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell)

Disadvantages:
 
- doesn't have a boatload of features that we don't need anyway
- can't easily introduce an ACE by accidentally using `load()` instead of `safe_load()`

`````markdown
````toml
[packages]
asdf = "<capella_pkg_uuid>"

[built_ins]
ros_pkg = "<capella_built_in_pkg_uuid>"

[custom_pkg]
abc = ["<capella_cls1_uuid>"]
xyz = [
  "<capella_cls2_uuid>",
  "<capella_cls3_uuid>",
]

[custom_types]
Bitset16 = "int16"
Bitset32 = "int32"
````
`````